tests/job/test_external.py

```diff
@@ -0,0 +1,38 @@
+import os
+import unittest
+
+from pyiron_base import Project
+
+job_py_source = """
+from pyiron_base import Notebook as nb
+coeff_tot = nb.get_custom_dict()
+print(coeff_tot['value'])
+"""
+
+class TestScriptJob(unittest.TestCase):
```

Comment:
While we're touching this it would be very nice to convert it over to subclass of `pyiron_base._tests.TestWithProject` and remove a bunch of the setup and teardown code.

Reply:
As we need the changes in the pull request rather urgently I would move that to a separate pull request

Reply:
It's five minutes. How urgent is urgent? Let me put my kid to bed and I'll do it (~20:15).

Reply:
I proposed the fix 4 hours ago https://github.com/pyiron/pyiron_base/pull/266  and we still need to create the conda package plus wait for the release to be available on the cluster. I guess the update of the test can also be done in a separate pull request and it is not required on the cluster.